src/main/java/eutros/multiblocktweaker/crafttweaker/construction/MultiblockBuilder.java

```diff
@@ -107,6 +108,19 @@ public MultiblockBuilder withTexture(@NotNull IICubeRenderer texture) {
         return this;
     }
 
+    /**
+     * Set the default zoom level for the multiblock.
+     * This is optional, and if not defined will default to 1.0f
```

Comment:
Doesn't this javadoc differ to the actual behaviour? `zoom` doesn't seem to get initialized otherwise.

Reply:
The default value comes from where it is initialized in GTCE. The value in MultiblockInfoPage defaults to 1.0f, and will only change if it is overridden.

Reply:
But it _is_ overridden, with `multiblock.zoom`, which is `0f` unless this method is called to set otherwise, surely?

Reply:
I have corrected this. The default value is now set to 1.0f unless the method is called in the builder to set it to a custom value.